modules/sagemaker/variables.tf

```diff
@@ -88,3 +88,8 @@ variable "config_bucket_key_arn" {
   description = "The ARN of the KMS key using which notebook scripts are encrypted in S3."
   type        = string
 }
+
+variable "preprocessing_script_path" {
```

Comment:
Is this supposed to be optional? If so, we should default to null. If the root level module passes 'null' to this variable, it will throw an error (in more recent versions of terraform)

Reply:
done